Calculator.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+
+public class CalculatorClass implements Operations{
+
+	public static void main(String[] args) {
+		
+		Scanner scan = new Scanner(System.in);
+		
+		double firstValue,secondValue,total=0;
+		
+		int operation;
+		
+		
+		System.out.println("Enter First Value");
+		firstValue = getInput(scan);
+		
+		System.out.println("Enter Second Value");
+		secondValue = getInput(scan);
+		
+		System.out.println("1)Sum \t2 2)Sub \n 3)Multiply \t 2)divide");
+		operation = (int)getInput(scan);
+		
+		scan.close(); //Close Input Stream
+		
+		switch(operation){
```

Comment:
Use of a While statement with an if nested would make more sense here and would make your life a bit easier.